scripts/globals/mobskills/ascetics_fury.lua

```diff
@@ -0,0 +1,26 @@
+---------------------------------------------
+--  Ascetics Fury
+--  Description: Hand To Hand Weapon Skill
+--  Type: Physical
+---------------------------------------------
```

Comment:
Please format these header areas like the BCNM files:
```lua
...
-- Horlais Peak Level Break
-----------------------------------
require("scripts/globals/battlefield")
require("scripts/globals/npc_util")
require("scripts/globals/quests")
require("scripts/globals/titles")
-----------------------------------

function onBattlefieldTick(battlefield, tick)
...
```

etc.

Reply:
Done, sorry for the extra spaces >.<